app/src/pages/inside/common/integrations/integrationHeader/integrationHeader.scss

```
@@ -65,45 +65,55 @@
+  @include integration-measure(13px, 20px, var(--rp-ui-base-e-400), var(--rp-ui-base-font-family));
 }
 
 .integration-description {
-  @include integrarion-measure(13px, 20px, $COLOR--almost-black, $FONT-ROBOTO-REGULAR);
+  @include integration-measure(
+    13px,
+    20px,
+    var(--rp-ui-base-almost-black),
+    var(--rp-ui-base-font-family)
+  );
   text-indent: 1px;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)